app/models/access_configuration.rb

```diff
@@ -1,19 +1,17 @@
+require 'digest/sha1'
+
 class AccessConfiguration < ActiveRecord::Base
   belongs_to :user
   store :service_options, coder: JSON
 
-  def self.get_default_options(user, cmr_concept_id)
-    access_config = self.find_by(user: user, dataset_id: cmr_concept_id)
-    if access_config
-      access_config.service_options
-    else
-      nil
-    end
+  def self.get_default_access_config(user, cmr_concept_id)
+    self.find_by(user: user, dataset_id: cmr_concept_id)
```

Comment:
the ```get_default_options``` used to return ```service_options```, not the whole ```access_config```. So the if-else is a nil check. It is removed because now the method returns the whole ```access_config```.